docs/insights/failure-analysis.md

```diff
@@ -17,6 +17,14 @@ Using Failure Analysis:
 Failure Analysis supports Playwright, Selenium and Appium. However, for Selenium and Appium to be effective, the tests must be configured to [report a pass/fail outcome](/basics/test-config-annotation/test-annotation#setting-passfail). Additionally, Appium is supported only for mobile web applications and not for native applications.
 :::
 
+## Where to access Failure Analysis 
```

Comment:
since we have such a huge product portfolio i would call out somewhere about this is for RDC and VDC only